src/creatorNtuples.ts

```diff
@@ -111,9 +111,6 @@ export default function creatorNtuples(
         header += `${xData[point]}, ${yData[point]}\n`;
       }
     } else if (options.isXYData) {
-      const firstX = xData[0];
-      const lastX = xData[xData.length - 1];
-      const deltaX = (lastX - firstX) / xData.length;
```

Comment:
Not sure if this error was anywhere else but it should have been devided by (xData.length-1) seems to me.